connexion/middleware/abstract.py

```diff
@@ -244,11 +244,10 @@ def __init__(self, app: ASGIApp) -> None:
         self.app = app
         self.apis: t.Dict[str, API] = {}
 
-    def add_api(
-        self, specification: t.Union[pathlib.Path, str, dict], **kwargs
-    ) -> None:
+    def add_api(self, specification: t.Union[pathlib.Path, str, dict], **kwargs) -> API:
         api = self.api_cls(specification, next_app=self.app, **kwargs)
         self.apis[api.base_path] = api
+        return api
```

Comment:
Just a fix to get the `AsyncApp` working correctly. Not related to enforcing defaults, but necessary to make the example work.